pymc3/tests/test_profile.py

```diff
@@ -23,7 +23,7 @@ def test_profile_model(self):
         assert self.model.profile(self.model.logpt).fct_call_time > 0
 
     def test_profile_variable(self):
-        assert self.model.profile(self.model.value_vars[0].logpt).fct_call_time > 0
+        assert self.model.profile(self.model.value_vars[0]).fct_call_time > 0
```

Comment:
This one does not seem equivalent... `self.model.value_vars` is just a dictionary call.

Reply:
It returns the `TensorVariable` which is then profiled, no?
Can you suggest a change? As you can see I'm still confused by these logpt things..

Reply:
I added a suggestion in my first comment, but it has to be tested locally, as I might have screwed up the names while copy pasting from the REPL